src/main/io/ledstrip.h

```diff
@@ -28,7 +28,7 @@
 
 #include "pg/pg.h"
 
-#define TASK_LEDSTRIP_RATE_HZ 50
+#define TASK_LEDSTRIP_RATE_HZ 60
```

Comment:
What is this change intended to solve?

Reply:
Not really affecting performance but might be result less inconsistent effect. As Larson run 60Hz and we only run the task on 50, there might be a skipped update. But it's not that important change.